Blockzilla/BrowserViewController.swift

```diff
@@ -958,8 +967,35 @@ extension BrowserViewController: URLBarDelegate {
     func urlBarDidTapShield(_ urlBar: URLBar) {
         showDrawer()
     }
+    
+    func urlBarDidLongPressUrl(_ urlBar: URLBar) {
+        let customURLItem = PhotonActionSheetItem(title: UIConstants.strings.customURLMenuButton, iconString: "icon_link") { action in
+            urlBar.addCustomURL()
+        }
+        let copyItem = PhotonActionSheetItem(title: UIConstants.strings.copyMenuButton, iconString: "icon_link") { action in
+            urlBar.copyToClipboard()
+        }
+        let pasteItem = PhotonActionSheetItem(title: UIConstants.strings.urlPaste, iconString: "icon_paste") { action in
+            urlBar.paste()
+        }
+        let pasteAndGoItem = PhotonActionSheetItem(title: UIConstants.strings.urlPasteAndGo, iconString: "icon_paste") { action in
```

Comment:
Hopefully we can get some different icons for these, let's follow up with Bram on this. Not a blocker for now, though.